.github/workflows/acceptance-test.yml

```diff
@@ -0,0 +1,17 @@
+name: "Acceptance Test"
+
+on:
+  schedule:
+  # Runs every day at midnight
+  - cron: "0 0 * * *"
+
+jobs:
+  acceptnace-test:
+    runs-on: ubuntu-latest
+    name: Acceptance Test
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.17'
+      - run: make testacc
```

Comment:
Example CI run of failing tests: https://github.com/hashicorp/packer/runs/5164743184. Need to follow-up to see if these are related to the CI changes or not

Reply:
Hi @mdeggies the failing don't appear to be related to the ci change. Tehy appear to be related to missing credentials for HCP and AWS. These keys are loaded from secrets directory in the packer-release repo,  which are exposed via environment variables. 

Are these variable that would would have to pull into the Packer's secrets or should they be managed in the common release tooling repo?

Reply:
Ahh, thanks for looking into this! For now, we'll need to port over the required secrets from packer-release into github repo level secrets in this project, and set the job to run on only merges to `master` or on a schedule (nightly or whatever makes sense for your team). In the future, this setup will change and tests will run within the common workflows repo.